charts/values.yaml

```diff
@@ -39,7 +39,9 @@ mayastor:
     node:
       initContainers:
         enabled: true
-
+  # etcd:
+  #   # -- Kubernetes Cluster Domain
+  #   clusterDomain: cluster.local
```

Comment:
Why not uncomment?

Reply:
In case mayastor chart moves to a different value, etc. There is no CI/automation to make the corresponding change on this chart. The behaviour we'd want out of this is: convey that this is configurable, but always fall back to the defaults issued by the mayastor chart.

Keeping it commented give us this flexibility. In case this value is updated on the mayastor chart, and this one is missed, then this chart would force the use of the outdated value and lead to undesirable behaviour.

Reply:
Do we need to add it here commented out at all?

Reply:
We haven't mentioned mayastor and the dependent charts clearly in the openebs.io/docs documentation. This yaml and this README.md may be only way for a naive user to find out about this.

Reply:
Adding to the README.md ok for the major settings, adding commented out to the yaml not sure.

Reply:
Sure I'll uncomment the configuration.